.editorconfig

```diff
@@ -149,6 +149,15 @@ dotnet_diagnostic.IDE0002.severity = warning
 # IDE0005: Remove unnecessary import
 dotnet_diagnostic.IDE0005.severity = warning
 
+# IDE0058: Remove unnecessary expression value
+dotnet_diagnostic.IDE0058.severity = none
+
```

Comment:
Potentially it can be good issue detector.

Reply:
There were around 60+ errors from different parts of the project with IDE0058. We could create a new issue to track them.